modules/aca/office_booking.rb

```diff
@@ -149,230 +73,78 @@ def on_update
         self[:booking_hide_all] = setting(:booking_hide_all) || false
```

Comment:
What's the different between `booking_hide_all` and `hide_all`?

Reply:
I believe `hide_all` was from a time when ngx-bookings (or the old AngularJS version of the booking panel frontend) was not using bindings under the `bookings_` namespace. I will remove hide_all.